web/models/package.ex

```diff
@@ -122,8 +122,7 @@ defmodule HexWeb.Package do
   def all_with_docs do
     from(p in Package,
          order_by: p.name,
-         where: not is_nil(p.docs_updated_at),
-         select: {p.name, p.docs_updated_at})
+         where: not is_nil(p.docs_updated_at))
```

Comment:
@ericmj I'm not sure what would be a good name for the 2nd function that's gonne be used by hex.pm/sitemap.xml, do you have any suggestions? Perhaps, instead `all_with_docs` we'd have something more specific, e.g.: `Package.packages_sitemap`, `Package.docs_sitemap`? Sorry for lots of questions, naming is hard :)